example/Makefile

```diff
@@ -13,12 +21,15 @@ image: $(image_tar)
 $(image_tar): Dockerfile service.py start.sh
 	docker run \
 		-v $(PWD):/workspace \
-		gcr.io/kaniko-project/executor:v1.9.2 \
+		gcr.io/kaniko-project/executor:v1.19.2 \
```

Comment:
is this bump necessary for the arch changes? If not, please move it to a separate PR.

Reply:
I thought it was needed, but just tested again and seems to work with the old version. I've reverted.